frontend/src/modules/auth/auth-routes.js

```diff
@@ -69,6 +70,12 @@ export default [
         component: InvitationPage,
         meta: { title: 'Invitation' },
       },
+      {
+        name: 'terms-and-privacy',
+        path: 'terms-and-privacy',
+        component: TermsAndPrivacyPage,
+        meta: { title: 'Terms of service and privacy policy' },
+      },
```

Comment:
Should we only allow this route if user is authenticated? Otherwise I'm able to access it, click on the button and get a 403 when not authenticated?